### PR TITLE
Add sample PR template

### PR DESCRIPTION
**Pull request by sciolkowski**
I think this is a good sample PR template (consists of a couple mixed together from Nu).
Of course, parts like commons, RN, API, Design and screenshots can be removed according to each PR needs. It will be nice to remove/add things that you guys find useful in your templates. Let's discuss it here.

Related to: https://github.com/HealthTechDevelopers/playbook/issues/33

### CODE REVIEW

**Comment by perzanko** [comment]
ref: https://github.com/HealthTechDevelopers/playbook/issues/33

**Comment by frasun** [comment]
@perzanko it is about the contents of PR template, not how to create one misio 🙂

**Comment by perzanko** [comment]
?
Screenshot 2019-06-05 at 15 32 45
Screenshot 2019-06-05 at 15 32 35

**Comment by perzanko** [comment]
?

Screenshot 2019-06-05 at 15 33 26
Screenshot 2019-06-05 at 15 34 04

**Comment by frasun** [comment]
@perzanko uhm sorry you are right, @sciolkowski this PR misses task reference 🙂

**Comment by frasun** [comment]
@sciolkowski could you make more generic? not every project has commons or API separated - if you could add sth like:
`If your project has dependencies mark them here e.g. ...`

What is your opinion on that?

**Comment by sciolkowski** [comment]
@frasun My opinion is that PR template can have additional sections which can be easily removed while making a PR. E.g. commons, RN checklist etc.

But if we would keep `If your project has dependencies mark them here e.g. ...` descriptions in the template itself, it would mean that even if you want to fill that section, you would have to next remove this description to make it look legit.

I know it's a sample and can be adjusted at the time it is added to the specific repo. I just assumed (maybe wrong 🤔), that it would end often as a copy-pasted one.

**Comment by frasun** [comment]
Ok, makes sense. Another thing is do we want to make the changlog section optional if attached commit list "speaks for itself"?

**Comment by sciolkowski** [comment]
@frasun That's a good question! Personally, even if IMO I pay attention to my commit messages I often fill this section with some more general or more detailed info depending on the situation, e.g.:

image
image

Sometimes I just decide that there are too many commits for the reviewer to go through and it would be nice to create such info-pill in the description. And sometimes there is one commit, but the solution requires additional explanation.

I don't feel attached to this section, I treat it as an option, but I use it often at the same time.

**Comment by maciej-zywno** [comment]
On a side, if we wish so we could have both some kind of a master pull request template that assumes a complex project where both screenshots, API changes, user flow and multitude other information should be included BUT add information like "Well, in real world you will most likely work in a project that might not need all of that so below we show a few examples of pull requests descriptions that are smaller yet still completely fine". My 2 cents :)

**Comment by sciolkowski** [comment]
Guys, I'll be glad for some rejections/approvals/suggestions but not listlessness 😭 ;)

For example, Maciek's suggestion seems to be reasonable, but the question is where would be the best place to put it? Another README inside `sources` folder? Another section in the workflow? I think the latter sounds more legit, but what are your thoughts?

**Comment by dskoneczny** [comment]
We can add a short description at the top of each PR template:
``` this is an example how PR template can look like and it should be customized to your project requirements/needs```
🤔 
or create README. in the directory with PR templates

**Comment by sciolkowski** [comment]
@dskoneczny Yeah, it's tempting to add it in the template itself but I'd try to avoid from reasons I state earlier in response to Paweł's suggestions. Having these two options I would go for an additional README. And basically, it's about the freedom to customize it even for each PR, not just project.

So I will try to gather some screenshots with very big or very small descriptions in PRs and push it here as a examples in the additional README file. Sounds like a good idea for now.

**Comment by jablonskipj** [comment]
What would you all tell about putting additional info about using the template as repository template (as suggested by @perzanko in the [#33](https://github.com/HealthTechDevelopers/playbook/issues/33))? Thanks to that the process of using the template would be automatic (I believe that this is the easiest way of achieving our purpose of using the template by all team members).

**Comment by pokorson** [comment]
@jablonskipj I believe we've already agreed on that? Templates in playbook will only serve as examples you can copy to your repo when kicking off the project

File: sources/sample_pr_template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+* One
+* Two
+* Three
+
+#### Tests:
+
+* [ ] Tests updated
+* [ ] Tests went green
+
+#### API Checklist:
```
Comment (pokorson) [inline sources/sample_pr_template.md]: what checklist is supposed to describe?

**Comment by dskoneczny** [comment]
@sciolkowski 
Example PR Template from the mobile application:
```
Task:
[JiraTaskNumber](https://htdevelopers.atlassian.net/browse/JiraTaskNumber)

### Quality checklist:
* [ ] Jest tests updated
* [ ] E2E tests updated

* [ ] **Tested on Android**
* [ ] **Tested on Old Iphones**
* [ ] **Tested on iPhones X**

### Test Cases
- ADD INFORMATION THAT CAN BE HELPFUL FOR QA

Changes:
- 

Screenshots/Gifs:
```

File: sources/sample_pr_template.md (hunk 1 above)
Comment (sciolkowski) [reply sources/sample_pr_template.md]: @pokorson Current version assumes that API documentation is kept in Postman, hence updated/new/removed endpoints should be reflected there. IMO it's enough, but we can add something if you find it valuable.

Comment (pokorson) [reply sources/sample_pr_template.md]: I think such information like sub-tasks done / to do should be rather kept in jira/trello. PR and work around it should be reviewed versus card requirements

Comment (sciolkowski) [reply sources/sample_pr_template.md]: The main purpose of these checks here is to remind about this stuff at the moment when PR is created, and I know from my experience that it helps :) And I also know that some of the teams base the whole flow on GH too, when such checks are very useful.

**Pull request merged**